Client/Frontend/Browser/Tab.swift

```diff
@@ -202,6 +202,20 @@ class Tab: NSObject {
         return lastTitle
     }
 
+    /// Use the display title unless it's an empty string, then use the base domain from the url
+    func getTabTrayTitle() -> String? {
+        let baseDomain = sessionData?.urls.last?.baseDomain ?? url?.baseDomain
+        var backUpName: String = "" // In case display title is empty
+
```

Comment:
Keeping this change to a minimum for v100 but we can add more fallback items as needed here. 

This should be a good starting point on addressing empty inactive tab title